### DISCUSSION TITLE
Why my Pull Shark contribution badge disappeared?

### DISCUSSION THREAD

**Discussion by jquanC** [community/community#37789] [Other Feature Feedback, Questions, & Ideas] [answered]
I got the Pull Shark contribution badge in August.
But today I found it disappeared. why is that? Does anyone know what's going on here？

**Comment by radar-blip** [comment, marked as answer]
Look here for it: https://github.com/jquanC?tab=achievements

If its there, click the eye icon to display it.

**Comment by jquanC** [comment]
Thanks！Lost little shark is back home.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "community/community"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月31日(星期一) 凌晨3:35
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [community/community] Why my Pull Shark contribution badge disappeared? (Discussion #37789)





 
Look here for it: https://github.com/jquanC?tab=achievements
 
If its there, click the eye icon to display it.
